docs/FAQ.md

```diff
@@ -35,6 +35,16 @@ Override `LD_PRELOAD` for that app. For flatpaks, this is as simple as removing
 3. Consider using distrobox or brew to install it
 4. Layer it using `rpm-ostree install`, as a last option
 
+#### How can I install command-line tools?
+
+[Homebrew](https://brew.sh/) is a package manager for Linux. It can be installed on secureblue by running `ujust brew`. Once installed, you can install packages with `brew install`. Homebrew installs packages in a separate home directory, but at the moment packages are not sandboxed and will have the same access as any other tool. You may be able to use `bubblejail` for sandboxing instead.
+
```

Comment:
This seems like a more elaborate version of the FAQ item above it? maybe we can combine them?